frontend/src/routes/docs/[...slug].svelte

```diff
@@ -1,22 +1,29 @@
-<script context="module">
-    export async function preload({ params: { slug } }) {
-        const response = await this.fetch(`process.env.FEATHER_API/docs/page/${encodeURI(slug)}`)
-        let markdown_html = await response.text();
-        return { html: markdown_html };
+<script context="module" lang="ts">
+    import * as docscache from '$stores/docscache';
+
+    export async function preload({ params: { slug } }: {params: {slug: string[]}}) {
+        let slugpath = slug.join("/");
+        if(docscache.has(slugpath)) {
+            return { html: docscache.get(slugpath) }
+        } else {
+            const response = await this.fetch(`process.env.FEATHER_API/docs/page/${encodeURI(slugpath)}`)
+            let markdown_html = await response.text();
+            markdown_html = markdown_html.replaceAll("http://localhost:3000", "/docs/");
```

Comment:
I think sapper only routes via JS if it is a relative link else the site would be refetched (also changed in _layout.svelte)